pkg/resources/custom-domain/prometheusRules.go

```diff
@@ -44,9 +44,39 @@ func customDomainCRErrorState(installationName string, namespace string, err str
 					"sop_url": resources.SopUrlRHOAMServiceDefinition,
 					"message": fmt.Sprintf(fmt.Sprintf("Error configuring custom domain, please refer to the documetaion to resolve the error. Found error: %s", err)),
 				},
-				Expr:   intstr.FromString(fmt.Sprintf("%s_custom_domain{customDomain='active'} > 0", installationName)),
+				Expr:   intstr.FromString(fmt.Sprintf("%s_custom_domain{active='false'} > 0", installationName)),
 				For:    "5m",
-				Labels: map[string]string{"severity": "warning", "product": string(v1alpha1.InstallationTypeManagedApi)},
+				Labels: map[string]string{"severity": "warning", "product": installationName},
+			},
+			{
+				Alert: "CustomDomainThreeScaleAdminUI",
```

Comment:
Small thing do we want to rename these alerts as they are not going to be only related to the custom domain work.

Reply:
Sure. What names would be more suitable?

Reply:
Trying to think of a name that is related to how the alert works, and they work by bypassing dns. So do we prefix them with `DnsBypassed`. In this example the alert name would be `DnsBypassedThreeScaleAdminUI`.